src/Models/BaseElement.php

```diff
@@ -1232,21 +1232,41 @@ public function getPageTitle()
     }
 
     /**
-     * @return boolean
+     * Returns true if this is the first element rendered in the ElementalArea
+     * @return bool
      */
-    public function First()
+    public function IsFirst(): bool
     {
         return ($this->Parent()->Elements()->first()->ID === $this->ID);
     }
 
     /**
-     * @return boolean
+     * @deprecated 5.4.0 Use IsFirst() instead.
      */
-    public function Last()
+    public function First()
+    {
+        Deprecation::notice('5.4.0', 'Use IsFirst() instead');
+        return $this->IsFirst();
+    }
+
+    /**
+     * Returns true if this is the last element rendered in the ElementalArea
+     * @return bool
```

Comment:
```suggestion
```
Sorry, should have caught this last time. We don't need the PHPDoc if it doesn't give any additional context over the actual type hint.